query_server/query/src/sql/parser.rs

```diff
@@ -1293,6 +1313,22 @@ impl<'a> ExtParser<'a> {
         }
     }
 
+    fn parse_change_node_state(&mut self) -> Result<ExtStatement> {
+        if self.parse_cnos_keyword(CnosKeyWord::NODE) {
+            let node_id = self.parse_number::<NodeId>()?;
```

Comment:
not match `[node_id]`
```
CHANGE NODE [node_id] STATE [Running | Cold | Pending]
```

Reply:
Why wouldn't it match? node_id is a number

Reply:
Syntax does not match code logic